src/lib.rs

```diff
@@ -77,11 +76,23 @@ impl WitBindgenVersion {
                 let bindgen_version = processors.iter().find_map(|(key, value)| {
                     key.starts_with("wit-bindgen").then(|| value.as_str())
                 });
-                match bindgen_version {
-                    Some(v) if v.starts_with("0.12.") => return Ok(Self::V0_12),
-                    Some(v) if v.starts_with("0.8.") => return Ok(Self::V0_8),
-                    Some(other) => return Ok(Self::Other(other.to_owned())),
-                    None => {}
+                if let Some(v) = bindgen_version {
```

Comment:
Could consider using https://crates.io/crates/semver for this.

Reply:
Seems a bit overkill for now, but if we run into issues I'll switch over.